exporters/otlp/src/otlp_grpc_exporter.cc

```diff
@@ -3,10 +3,11 @@
 
 #include "opentelemetry/exporters/otlp/otlp_grpc_exporter.h"
 #include "opentelemetry/exporters/otlp/otlp_recordable.h"
+#include "opentelemetry/sdk/common/global_error_handler.h"
```

Comment:
Again, you are forcing the code to know of the global error handler. It does not need to know. It'd only need to include `config.h` , which should then give the rest of details. I think `config.h` is already included at this point to define the `OPENTELEMETRY_` macros, just move it in there.

Reply:
Good point. Will change that.